src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php

```diff
@@ -255,43 +254,108 @@ protected function addIns(array $inFields, array $inValues)
      * adds where statements for in-clauses
      * @param array $betweenFields
      * @param array $betweenValues
+     * @param array $betweenConjunctions
      */
-    protected function addBetweens(array $betweenFields, array $betweenValues)
+    protected function addBetweens(array $betweenFields, array $betweenValues, array $betweenConjunctions)
     {
         $betweenParts = array();
+        $firstConjunction = null;
+
         foreach ($betweenFields as $betweenField) {
-            $betweenParts[] = $betweenField->getSelect() .
-                ' BETWEEN :' . $betweenField->getName() . '1'.
+            $conjunction = ' ' . $betweenConjunctions[$betweenField->getName()] . ' ';
+
+            if (!$firstConjunction) {
```

Comment:
Instead of this logic I would just save the values in `$betweenParts`, and use implode on it later.